### DISCUSSION TITLE
Programs on modrinth

### DISCUSSION THREAD

**Discussion by IZeeXeS** [modrinth/.github#60] [Feature requests]
Add a page for programs.

Minecraft related programs typically scattered around many sites, it would be useful for me and others to have a single place for it

**Comment by ghost** [comment]
out of scope for modrinth sadly

**Comment by triphora** [comment]
Not really. This is the sort of thing Project Ogre is meant to address

**Comment by ghost** [comment]
What is that?

**Comment by heipiao233** [comment]
Programs? Like NBTEditor?

**Comment by MMK21Hub** [comment]
Yeah, I assume that this refers to tools that operate outside of Minecraft but interact in Minecraft in some way. I'm not sure how they plan on presenting them in a coherent and organised manner, though.

**Comment by quat1024** [comment]
There's some prior art for this; [Spigot's platform](https://www.spigotmc.org/resources/categories/standalone.19/) has a section for standalone programs

**Comment by RECHERGG** [comment]
upvote
